firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -51,29 +51,38 @@ export class FirestoreBigQueryEventHistoryTracker
 
   async record(events: FirestoreDocumentChangeEvent[]) {
     await this.initialize();
-
+    const options = {
+      raw: true,
```

Comment:
It tells the BigQuery streaming API to expect the payload we're sending instead of plain row data. https://googleapis.dev/nodejs/bigquery/latest/Table.html#insert.

Reply:
Got it, thanks!